jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/AuthenticationProtocolHandler.java

```diff
@@ -238,7 +238,7 @@ public void onComplete(Result result)
             catch (Throwable x)
             {
                 if (LOG.isDebugEnabled())
-                    LOG.debug("Authentication failed", x);
+                    LOG.atDebug().setCause(x).log("Authentication failed");
```

Comment:
Not necessary here, you are in a catch block already.
The Throwable will exist.